doc/makeConnected/pgr_makeConnected.rst

```diff
@@ -112,6 +112,12 @@ Column           Type        Description
 Additional Example:
 -------------------------------------------------------------------------------
 
+Using pgr_connectedComponents() to check the number of connected components in the graph.
```

Comment:
Actually, make connected has more meaning with combination of connected components
I prefer to not have an additional example unless it is absolutely necessary

Reply:
Okay, I will remove the additional example. In the main example pgr_connectedComponents() is sufficient.